src/app/pcasts/controllers/get_change_user_by_id_controller.py

```diff
@@ -0,0 +1,22 @@
+from . import *
+
+class GetUserByIdController(AppDevController):
```

Comment:
We should change the name of this controller now - `UserCRUDController` would make more sense.